docs/design/fsdp-design-en.md

```diff
@@ -0,0 +1,373 @@
+# DLRover's FSDP Elastic Training Design Document
+
+## 1. Background
+
+Large-scale machine learning models have expanded, necessitating
```

Comment:
necessitating -> necessitated ?

Reply:
Thanks for pointing it out. I've revisited the phrasing and made refinements. The mentioned issues should be resolved now in the latest commit.